utils/get_ci_error_statistics.py

```diff
@@ -88,23 +113,26 @@ def get_errors_from_single_artifact(artifact_zip_path):
             " problem."
         )
 
-    return errors, failed_tests
+    job_link = None
+    if job_name and job_links:
+        job_link = job_links.get(job_name, None)
+
+    # A list with elements of the form (line of error, error, failed test)
+    result = [x + [y] + [job_link] for x, y in zip(errors, failed_tests)]
```

Comment:
Add job links, so we can go the job run pages in order to see more information about the errors.